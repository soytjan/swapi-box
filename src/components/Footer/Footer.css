footer {
  color: white;
  height: 80px;
  display: flex;
  justify-content: center;
  align-items: center;
  padding: 20px 0 0;
  font-weight: 400; }
