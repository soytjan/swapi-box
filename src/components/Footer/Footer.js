import React from 'react';
import './Footer.css';

const Footer = () => {
  return (
    <footer>
      <p>&copy; 2018 Amanda Tjan</p>
    </footer>
  )
}

export default Footer;