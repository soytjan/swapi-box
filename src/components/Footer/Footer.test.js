import React from 'react';
import Footer from './Footer';
import { shallow } from 'enzyme';

describe('Footer', () => {
  it('matches snapshot', () => {
    const renderedComponent = shallow(<Footer />)

    expect(renderedComponent).toMatchSnapshot();
  })
})