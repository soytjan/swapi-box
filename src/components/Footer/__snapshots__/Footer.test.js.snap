// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Footer />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <p>
        © 2018 Amanda Tjan
</p>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "© 2018 Amanda Tjan",
      },
      "ref": null,
      "rendered": "© 2018 Amanda Tjan",
      "type": "p",
    },
    "type": "footer",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <p>
          © 2018 Amanda Tjan
</p>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "© 2018 Amanda Tjan",
        },
        "ref": null,
        "rendered": "© 2018 Amanda Tjan",
        "type": "p",
      },
      "type": "footer",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
